SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_MaterialFarmerTools.h

```diff
@@ -24,6 +24,28 @@ namespace PokemonAutomation{
 namespace NintendoSwitch{
 namespace PokemonSV{
 
+
+class MaterialFarmerTools {
+public:
+    struct Stats : public LetsGoEncounterBotStats{
```

Comment:
You don't need to wrap it in a class. It's no different than any other class so it can sit outside.